src/routes/Bookings/getOfferrerVenues.ts

```diff
@@ -0,0 +1,21 @@
+import * as pcapi from 'repository/pcapi'
+import { formatAndOrderVenues } from 'repository/venuesService'
```

Comment:
Si on suit l'idée que j'ai posé plus haut, on pourrait avoir 
```suggestion
import { formatAndOrderVenues } from 'core/venues/utils
```

Reply:
oui c'est pas mal de regrouper ce genre de choses, le terme `core` me plait moyen dans ce contexte mais l'idée est à creuser.